lib/card.rb

```diff
@@ -1 +1,9 @@
-# Good luck!
+class Card
+  attr_reader :question, :answer
+  attr_accessor :deck
```

Comment:
I think for something like this I'd prefer a separate method as opposed to an `attr_accessor`